apps/finance/app/src/lib/date-utils.js

```diff
@@ -0,0 +1,8 @@
+import { format } from 'date-fns'
+
+export const MMDDYY_FUNC_FORMAT = 'MM/dd/yy'
+export const YYMMDD_LONG_FORMAT = 'yyyy-MM-dd'
+
+export function formatDate(date) {
```

Comment:
Not sure if there are other areas where we use `format()` directly, but we could settle on always importing `formatDate()` from this `date-utils` when we need to format dates.

Then we can add the format we'd like as a parameter, and default it to `MMDDYY_FUNC_FORMAT`.